lametro/utils.py

```diff
@@ -28,37 +29,50 @@ def parse_subject(text):
 
     return None
 
-# Use this helper function when multiple "current meetings" happen simultaneously.
+
 def calculate_current_meetings(found_events, now_with_buffer):
-    earliest_start = found_events.first().start_time
-    latest_start = found_events.last().start_time
-    # Sometimes multiple events happen at the same time (though in reality, they occur one-after-the-other). Example: the committee events at 1:00 on 05/17/2017.
-    # Check for this situation, and then determine if these events include a board meeting.
-    # If current events do not include a board meeting, the end time should be one hour.
-    if earliest_start == latest_start:
-        if found_events.filter(name__icontains='Board Meeting'):
-            # Custom order: show the board meeting first.
-            # '.annotate' adds a field called 'val', which contains a boolean – we order in reverse, since
-            # false comes before true.
-            return found_events.annotate(val=RawSQL("name like %s", ('%Board Meeting%',))).order_by('-val')
-        else:
-            one_hour_ago = now_with_buffer - timedelta(hours=1)
-            
-            return found_events.filter(start_time__gt=one_hour_ago)
-
-    else:
-        # To find committee events...
-        event_names = [e.name for e in found_events if ("Committee" in e.name) or ("LA SAFE" in e.name) or ("Budget Public Hearing" in e.name) or ("Fare Subsidy Program Public Hearing" in e.name) or ("Crenshaw Project Corporation" in e.name)]
-
-        if event_names:
-            # Set meeting time to one hour
-            one_hour_ago = now_with_buffer - timedelta(hours=1)
-            
-            found_events = found_events.filter(start_time__gt=one_hour_ago)
-
-            if len(found_events) > 1:
-                return found_events
-            else:
-                return found_events.first()
-        else:
-            return found_events.first()                
\ No newline at end of file
+    # Metro provided a spreadsheet of average meeting times. The minimum average time a meeting lasts is 52 minutes: let's round to 55 and remove the 5-minute buffer. 
```

Comment:
Does that mean `five_minutes_from_now` in the `current_meeting` should just be now?

Reply:
Well, Metro still wants a meeting to appear fives minutes before its start time...so, actually my comment should say: "and add the 5-minute buffer."